lib/zold/wallets.rb

```diff
@@ -32,11 +32,19 @@ def initialize(dir)
       @dir = dir
     end
 
-    # @todo #70:30min Let's make it smarter. Instead of returning
-    #  the full path let's substract the prefix from it if it's equal
-    #  to the current directory in Dir.pwd.
+    # Returns wallets directory path
     def to_s
-      path
+      dir = Dir.pwd.sub(path_start, '').split('/')
```

Comment:
@mulev the code here should really be extracted to a library. Violates [SRP](https://en.wikipedia.org/wiki/Single_responsibility_principle).

Also, it's too low-level, hard to reason about and violates [SLA](http://principles-wiki.net/principles:single_level_of_abstraction).